extensions/amp-live-list/amp-live-list.md

```diff
@@ -323,6 +323,21 @@ The `amp-live-list` exposes the following actions you can use [AMP on-syntax to
   </tr>
 </table>
 
+The `amp-live-list` also generates the following events for which you can use [AMP on-syntax to listen](../../../src/spec/amp-actions-and-events.md):
+
+<table>
+  <tr>
+    <th>Event</th>
+    <th>Description</th>
+    <th>Data</th>
+  </tr>
+  <tr>
+    <td>update</td>
+    <td>Fired when the list's DOM items are inserted or replaced</td>
+    <td><code>event.insertItemIds</code> : the <code>id</code>s of inserted list items, <code>event.replaceItemIds</code> : the <code>id</code>s of replaced list items</td>
```

Comment:
Would it be more accurate to call the event data fields insert**ed**ItemIds and replac**ed**ItemIds?